plugins/ibm_qradar/help.md

```diff
@@ -0,0 +1,1068 @@
+|filter|string|None|False|This parameter is used to restrict the elements in a list base on the contents of various fields|None|id=100|
+|offense_id|integer|None|True|The ID of the offense to get its notes|None|100|
+|range|string|None|False|Use this parameter to restrict the number of elements that are returned in the list to a specified range. The list is indexed starting at zero|None|1-2|
+
+Example input:
+
+```
+{
+  "fields": "id",
+  "filter": "id=100",
+  "offense_id": 100,
+  "range": "1-2"
+}
+```
+
+##### Output
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|data|[]note|False|JSON Data of the Offense Notes|
+
+Example output:
+
+```
+{
+  "data": [
+    {
+      "create_time": 1641385946573,
+      "id": 159,
+      "note_text": "New Note text",
+      "username": "API_user: admin"
+    },
+    {
+      "create_time": 1641451278329,
+      "id": 209,
+      "note_text": "New Note text",
+      "username": "API_user: admin"
+    }
+  ]
+}
+```
+
+#### Get Offense Notes by Id
+
+This action is used to get Offense Notes by Id.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|fields|string|None|False|Use this parameter to specify which fields you would like to get back in the response. Fields that are not named are excluded. Specify subfields in brackets and multiple fields in the same object are separated by commas|None|id|
+|note_id|integer|None|True|The ID of the offense note to get|None|100|
+|offense_id|integer|None|True|The ID of the offense to get its notes|None|100|
+
+Example input:
+
+```
+{
+  "fields": "id",
+  "note_id": 100,
+  "offense_id": 100
+}
+```
+
+##### Output
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|data|note|False|JSON Data of the Offense Notes for given id|
+
+Example output:
+
+```
+{
+  "data": {
+    "create_time": 1640936459993,
+    "id": 51,
+    "note_text": "Test ",
+    "username": "API_user: admin"
+  }
+}
+
+```
+
+#### Get Offenses
+
+This action is used to list all Offenses.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|fields|string|None|False|Use this parameter to specify which fields you would like to get back in the response. Fields that are not named are excluded. Specify subfields in brackets and multiple fields in the same object are separated by commas|None|id|
+|filter|string|None|False|This parameter is used to restrict the elements in a list base on the contents of various fields|None|id=1000|
+|range|string|None|False|Use this parameter to restrict the number of elements that are returned in the list to a specified range. The list is indexed starting at zero|None|1-2|
+|sort|string|None|False|Use parameter to sort the elements in a list|None|+id|
+
+Example input:
+
+```
+{
+  "fields": "id",
+  "filter": "id=1000",
+  "range": "1-2",
+  "sort": "+id"
```

Comment:
Do you need + for the field Name. Also, can you do sorting on multiple fields. If so, kindly include the examples.

Reply:
Yes, we need + or - sign before the field name. As it indicates the order or sorting + indicates the ascending order and - indicates the descending sort.